### PR TITLE
dev-python/numpy: version bump to 1.11.0

### PR DESCRIPTION
**Pull request by g-braeunlich**
Gentoo-Bug: 580790
Also bump to EAPI version 6

Package-Manager: portage-2.2.28

### CODE REVIEW

**Comment by kensington** [comment]
@gentoo/sci @gentoo/python

File: dev-python/numpy/numpy-1.11.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
+PYTHON_REQ_USE="threads(+)"
+
+FORTRAN_NEEDED=lapack
+
+inherit distutils-r1 flag-o-matic fortran-2 multilib multiprocessing toolchain-funcs versionator
```
Comment (SoapGentoo) [inline dev-python/numpy/numpy-1.11.0.ebuild]: `multilib.eclass` is not needed anymore with `EAPI=6` for `get_libdir()`

File: dev-python/numpy/numpy-1.11.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+	# don't version f2py, we will handle it.
+	sed -i -e '/f2py_exe/s: + os\.path.*$::' numpy/f2py/setup.py || die
+
+	# we don't have f2py-3.3
+	sed \
+		-e 's:test_f2py:_&:g' \
+		-i numpy/tests/test_scripts.py || die
+
+	distutils-r1_python_prepare_all
+	eapply_user
```
Comment (SoapGentoo) [inline dev-python/numpy/numpy-1.11.0.ebuild]: `distutils-r1_python_prepare_all` already calls `default` in `EAPI=6`, hence a separate `eapply_user` shouldn't be necessary.

**Comment by g-braeunlich** [comment]
OK. Applied your comments.

**Pull request closed without merging**